source/settings/settings-reference.txt

```diff
@@ -0,0 +1,103 @@
+.. _compass-interface-settings:
+.. _compass-settings-reference:
+
+--------------------------
+Compass Interface Settings
+--------------------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page provides an overview on options that you can set on the 
+|compass| interface. For more detailed guidance on how to set a specific option, 
+see the task page for the specific option.
```

Comment:
[suggestion] I was a little confused about _how_ to set these options until I got to the bottom of the page. Maybe it would make sense to say: 

"... needs. Use the [command line](url) or a [configuration file](url) to set the options you choose. Some options are only available in the [Settings panel](url)."

Reply:
Ah thanks for pointing this confusion out — I totally overlooked this. 😅

One thing to note is that this page is supposed to be a reference page specifically for settings you can only set on the Compass Settings UI panel, so will likely adjust the copy to be more specific to that.